packages/dm-core-plugins/src/explorer/style.css

```diff
@@ -0,0 +1,3 @@
+body .lm_content {
```

Comment:
this will make golden layout panels scrollable

example in https://github.com/golden-layout/golden-layout/issues/29